hyperspy/signal.py

```diff
@@ -3169,6 +3171,142 @@ def integrate_simpson(self, axis, out=None):
             return s
     integrate_simpson.__doc__ %= (ONE_AXIS_PARAMETER, OUT_ARG)
 
+    def fft(self, shifted=False, **kwargs):
+        """Compute the discrete Fourier Transform.
+
+        This function computes the discrete Fourier Transform over the signal
+        axes by means of the Fast Fourier Transform (FFT) as implemented in
+        numpy.
+
+        Parameters
+        ----------
+        shifted : bool, optional
+            If True, the origin of FFT will be shifted in the centre (Default: False).
+
+        **kwargs
+            other keyword arguments are described in np.fft.fftn().
+
+        Return
+        ------
+        s : ComplexSignal
+
+        Examples
+        --------
+        >>> im = hs.signals.Signal2D(scipy.misc.ascent())
+        >>> im.fft()
+        <Image, title: FFT of , dimensions: (|512, 512)>
+
```

Comment:
Could you add the power spectrum example from the user guide to the docstring example as well? Since most people using this function will probably expect the same output as they get in DM.